joins/README.md

```diff
@@ -0,0 +1,56 @@
+## JOINS track
+
+This track contains an artificial dataset intended to test JOIN operations with different key cardinalities.
+
+The dataset can be generated using the scripts in the `_tools` directory.
+
+### Example Documents
+
+Main index: 
+
+```json
+{
+  "id": 56,
+  "@timestamp": 946728056,
+  "key_1000": "56",
+  "key_100000": "56",
+  "key_200000": "56",
+  "key_500000": "56",
+  "key_1000000": "56",
+  "key_5000000": "56",
+  "key_100000000": "56",
+  "field_0": "text with value 0_56",
+  "field_1": "text with value 1_56",
+  "field_2": "text with value 2_56",
+  ...
+  "field_99": "text with value 99_56"
+}
+```
+
+The cardinality of the keys is the same as the key name, eg. `key_1000` will have 1000 different values in the dataset,
+from `0` to `999`.
```

Comment:
This assumes the file was generated with at least 1k documents.
This is likely true for small cardinality fields, but might not be the case for 100m.
I wonder if it is worth mentioning explicitly?

Reply:
👍 I added a small note